azure-cron.yml

```diff
@@ -68,7 +68,7 @@ jobs:
           # >= 0.13.43 are built this way.
           for version in $(echo $RELEASES | sed -e 's/ /\n/g'); do
             git checkout "$version"
-            while ! nix-build -A tools -A cached nix; do; done
+            while ! nix-build -A tools -A cached nix; do:; done
```

Comment:
Is that how a while loop in bash looks like?

Reply:
`:` is a built-in version of `/bin/true`; it's a no-op.